<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cooper Black</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/1.jpeg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showtime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Segoe UI Variable Small Semibol&quot;;
border-image: url(:/new/prefix1/labels/11.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>321</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="timeone">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>06:00-08:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timetwo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>08:00-10:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timethree">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10:00-12:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timefour">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>12:00-14:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timefive">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>14:00-16:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timesix">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16:00-18:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timeseven">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>18:00-20:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timeeight">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>20:00-22:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timenine">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>22:00-24:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>227</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="timerecord">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>计时器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>计划-记录 比较</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeBackgroundButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更换背景图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fontButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更换全局字体</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeColorAction">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更换背景颜色</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
